### DISCUSSION TITLE
Understanding the relationship between the schedule property in the root and the schedule property in the packageRules

### DISCUSSION THREAD

**Discussion by paulzakin** [renovatebot/renovate#16282] [Request Help] [answered]
After reading the documentation, I have a lingering question about scheduling behaviour.

Let's say we have the following `renovate.json`.

```json
{
  "extends": ["config:base"],
  "schedule": ["schedule:daily"],
  "packageRules": [
    {
      "matchPackagePatterns": ["*"],
      "matchUpdateTypes": ["minor", "patch"],
      "groupSlug": "all-minor-patch",
      "schedule": ["before 2am on monday"]
    },
    {
      "matchPackagePatterns": ["*"],
      "matchUpdateTypes": ["major"],
      "groupSlug": "all-major",
      "schedule": ["before 2am on tuesday"]
    }
  ]
}
```

Does the scheduling work like this?

1. Every day, the bot itself runs.
2. The bot scans the packageRules array.
3. If it is a Monday, the bot runs the `all-minor-patch` group. If it is a Tuesday, the bot runs the `all-major` group.

Then, let's say we change the root `schedule` property to "schedule:weekly", so it only runs every Monday. 

Does that mean only `all-minor-patch` will ever run? After all, `all-major` runs only on Tuesday, and the Renovate bot itself doesn't run? Or is it smart enough to get around that?

**Comment by viceice** [comment, marked as answer]
the package rule will override the root schedule

**Reply by paulzakin** [reply]
Ok awesome, thanks!
